unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/WearableGridItemComponentView.cs

```diff
@@ -51,14 +74,27 @@ public override void Awake()
                     else
                         OnEquipped?.Invoke(model);
                 }
-                else
-                    OnSelected?.Invoke(model);
```

Comment:
how is that `OnSelected` triggered now?

Reply:
In the awake, in the Click/Double click handling